lnwallet/interface_test.go

```diff
@@ -771,6 +771,7 @@ func testListTransactionDetails(miner *rpctest.Harness,
 		t.Fatalf("unable to fetch tx details: %v", err)
 	}
 
+	blockTxOuts := make(map[chainhash.Hash]map[chainhash.Hash][]*wire.TxOut)
```

Comment:
This could use a comment explaining that the mapping is from: blockHash -> transactionHash -> transactionOutputs.